cdc/log.cc

```diff
@@ -216,6 +216,9 @@ class cdc::cdc_service::impl : service::migration_listener::empty_listener {
     }
 
     void on_before_drop_column_family(const schema& schema, std::vector<mutation>& mutations, api::timestamp_type timestamp) override {
+        if (schema.is_view()) {
+            return;
+        }
```

Comment:
Why is this extra check necessary? Won't we just realize that the view does _not_ have a CDC log, and do nothing? Did anything fail without this extra check?

Reply:
I didn't test specifically with CDC and so I didn't want to change the flow for CDC.
Maybe this is worth at least a FIXME comment and an issue opened about this to consider removing this if statement if everything works without it, but that said, it's legitimate to consider that views do not have CDC logs and short-circuit their handling on this path.

Reply:
fixed in next version

Reply:
What was fixed?

Reply:
oh, I must have replied on the wrong thread, sorry.
nothing changed here indeed.

Reply:
removed this condition in v4.